packages/cli/templates/defaults/__templates__/route-template.tsx

```diff
@@ -94,7 +95,7 @@ export const meta: V2_ServerRuntimeMetaFunction<typeof loader> = ({ data }) => {
   if (data === undefined) {
     return metas;
   }
-  const { pageMeta } = data;
+  const { pageMeta, projectMeta } = data;
```

Comment:
maybe move back to client side instead of sending with loader?

Reply:
Have feeling that it will be a function like pageMeta too